test/io/test_io.cpp

```diff
@@ -1365,6 +1366,36 @@ TEST (PCL, Locale)
 #endif
 }
 
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+TEST (PCL, AutoIO)
```

Comment:
Since you're working with a ```PointCloud<PointT>``` type, i would recommend you write a typed test case like [this](https://github.com/renatoGarcia/pcl/blob/0a0bdb9c7e146583a7ed32230e6f28f5a2ee84d4/test/io/test_io.cpp#L1017-L1020) but for all point types. The macro is ```PCL_POINT_TYPES```.